hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -48,38 +50,44 @@
 (def pid-file (str dir "/server.pid"))
 (def log-file (str dir "/server.log"))
 
+(def CLIENT_ID_SEPARATOR "xxxxxxxxxx")
+(def REENTRANT_LOCK_ACQUIRE_COUNT 2)
+(def NUMBER_OF_PERMITS 2)
+(def LOGGING_SLEEP_DURATION_MS 1000)
+(def INVALID_FENCE 0)
+
 (defn build-server!
   "Ensures the server jar is ready"
   [test node]
   (when (= node (jepsen/primary test))
     (when-not (.exists (io/file local-server-jar))
-      (info "Building server")
-      (let [{:keys [exit out err]} (sh "lein" "uberjar" :dir "server")]
-        (info out)
-        (info err)
-        (info exit)
-        (assert (zero? exit))))))
+    (info "Building server")
+    (let [{:keys [exit out err]} (sh "lein" "uberjar" :dir "server")]
+      (info out)
+      (info err)
+      (info exit)
+      (assert (zero? exit)))))
+ )
```

Comment:
It looks like this might be mis-formatted--did you mean to take this back a level of indentation?